Mesh_3/include/CGAL/Mesh_3/Protect_edges_sizing_field.h

```diff
@@ -444,7 +444,13 @@ class Protect_edges_sizing_field
       if(dim == 0) msg << "corner (";
       else msg << "point (";
       msg << p << ")";
-      CGAL_error_msg(msg.str().c_str());
+      CGAL_error_msg(([this, str = msg.str()](){
+        CGAL_USE(this);
+#if CGAL_MESH_3_PROTECTION_DEBUG & 4
```

Comment:
@maxGimeno could you please check why this results in the following [error](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.1-Ic-100/Generator/TestReport_afabri_x64_Cygwin-Windows10_MSVC2017-Release-64bits.gz)

Reply:
My bad. One should not use preprocessor directive inside the arguments of a macro. That is not portable.

Reply:
Fixed by https://github.com/CGAL/cgal/pull/4074/commits/6d215c776bde45f59f3afb386aea4372a0777a2f.